server/src/controllers/loan.controllers.ts

```diff
@@ -0,0 +1,53 @@
+import express, { type Request, type Response } from "express";
+import { read } from "fs";
+import OutstandingLoan, {Loan} from "../models/OutstandingLoan";
+const mongoose = require("mongoose");
+
+
+const createOutstandingLoan = async (req: Request, res: Response) => {
+    try {
+        const {
+            initialPayment,
+            initialPaymentDate,
+            principalLeft,
+            nextPaymentDate,
+            nextPaymentAmount,
+            archivedLoan,
+            beneficiary,
+            validLoan
+        } = req.body;
+
+        
+        const newLoan = await OutstandingLoan.create(req.body as Loan);
+        await newLoan.save();
+
+        return res.status(200).json(newLoan);
+    } catch (err) {
+        console.error(err);
```

Comment:
Please handle all errrors in typesafe way. See example here: 
https://github.com/ChangePlusPlusVandy/vac-net/blob/a9c4e9bab9f8ff75490e1b4e561bcb6290a39fbc/server/src/controllers/session.controllors.ts#L19